server/lib/config.py

```diff
@@ -27,6 +27,7 @@
     'custom',
     'feedingamerica',
     'stanford',
+    'stanford-staging',
```

Comment:
nit: Sorry for not catching this earlier, but can we rename this staging-stanford (to match local-* instances below?)

Reply:
Well, I think that would break the matching with the GCP project name which is datcom-stanford-staging :(

Reply:
Ah ok - no worries let's keep it then.